hackathon_site/registration/jinja2/registration/application.html

```diff
@@ -69,8 +70,12 @@ <h4>Questions</h4>
     {{ render_field(form.q1, show_help_text_as_label=True) }}
     {{ render_field(form.q2, show_help_text_as_label=True) }}
     {{ render_field(form.q3, show_help_text_as_label=True) }}
+    {{ render_field(form.q4, show_help_text_as_label=True) }}
```

Comment:
You also need to set the appropriate classes in `registration/forms.py` for these fields, otherwise they look terrible. Copy what is there for q1-q3, adjusting the `data-length` if necessary